1.8/usage/haproxy.md

```diff
@@ -0,0 +1,77 @@
+---
+post_title: Configuring HTTP Proxy in Front of DC/OS
+nav_title: Configure HTTP Proxy
+menu_order: 61
+---
+
+You can configure a secure communication channel between HAProxy and the DC/OS Admin Router, using a custom hostname and port.
```

Comment:
The instructions here hold true for both, Enterprise DC/OS as well as Open DC/OS (the only difference being where to obtain the CA certificate from, which is a problem more-or-less unrelated to the problem that this docs page here is supposed to address). In order to reduce maintenance efforts, let's try to not split this up. I submit another review for suggesting corresponding changes.